src/arco_era5/pangeo.py

```diff
@@ -60,7 +68,8 @@ def run(make_path: t.Callable[..., str], date_range: t.List[datetime.datetime],
         'chunks': {'time': 4},
     }
 
-    fs = gcsfs.GCSFileSystem(project=os.environ.get('PROJECT', 'ai-for-weather'))
+    fs = gcsfs.GCSFileSystem(project=os.environ.get('PROJECT', 'ai-for-weather')) \
+                                if not parsed_args.local_run else LocalFileSystem()
```

Comment:
```suggestion
    fs = (
        gcsfs.GCSFileSystem(project=os.environ.get('PROJECT', 'ai-for-weather'))
        if not parsed_args.local_run else LocalFileSystem()
    )
```

I prefer this fluent interface for ternary expressions.

Reply:
necessary changes are done.